tslint.json

```diff
@@ -69,6 +70,54 @@
             "check-separator",
             "check-type"
         ],
-        "no-string-literal": true
+
+        // TODO
+        "arrow-parens": false, // [true, "ban-single-arg-parens"]
+        "arrow-return-shorthand": false,
+        "ban-types": false,

```

Comment:
do not think we will do that one.

Reply:
This would prevent us from typing a function as `Function`, which is unsafe. It also seems to fail on every reference to `Symbol` since we shadow that, but could be configured to ignore that one.

Reply:
Wasn't too hard to fix. #19586